emgapianns/management/commands/import_summary.py

```diff
@@ -42,43 +51,31 @@ def _check_source_file(source_file):
         :return:
         """
         if not os.path.exists(source_file):
-            logger.error('Path %r does not exist. SKIPPING!' % source_file)
+            logger.error("Path %r does not exist. SKIPPING!" % source_file)
             return False
         if not os.path.isfile(source_file):
-            logger.error('Specified source %r is not a file. SKIPPING!' % source_file)
+            logger.error("Specified source %r is not a file. SKIPPING!" % source_file)
             return False
         if os.stat(source_file).st_size == 0:
-            logger.error('Specified input file %r is empty. SKIPPING!' % source_file)
+            logger.error("Specified input file %r is empty. SKIPPING!" % source_file)
             return False
         return True
 
     def find_path(self, obj, options):
-        rootpath = options.get('rootpath', None)
-        self.suffix = options.get('suffix', None)
+        rootpath = options.get("rootpath", None)
+        self.suffix = options.get("suffix", None)
 
         if self.suffix in self.suffixes:
-            # Output names and folders changed slightly between v4 and v5
-            # This piece of code still supports both versions
-            for infix in ['%s_summary%s', '%s.summary%s']:
-                name = infix % (obj.input_file_name, self.suffix)
-                sub_dir = self.FUNCTION_SUB_DIR
-                if self.suffix in self.pathway_suffixes:
-                    sub_dir = self.PATHWAY_SUB_DIR
-                source_file = os.path.join(rootpath, obj.result_directory, sub_dir, name)
-                if not self._check_source_file(source_file):
-                    logger.info('Could not find: %s' % source_file)
-                else:
-                    logger.info('Found: %s' % source_file)
-                    self._parse_and_load_summary_file(source_file, obj)
-                    break
-        elif self.suffix in self.kegg_pathway_suffix:
-            name = '%s.summary%s' % (obj.input_file_name, self.suffix)
-            source_file = os.path.join(rootpath, obj.result_directory, self.PATHWAY_SUB_DIR, name)
+            name = "%s_summary%s" % (obj.input_file_name, self.suffix)
+            source_file = os.path.join(rootpath, obj.result_directory, name)
```

Comment:
The way you simplified this assumes that obj.result_directory resolves the correct sub directory path, which think it doesn't.
For instance the antismash GFF file resolves to the following sub directory: ERZXXXXXX_FASTA/pathways-systems

Reply:
This is a mistake, I accidentally overwrote the dependency-version-upgrades changes. I'll restore the file and add the changes from genome properties.